Jul 28 18:44:53.59
GET
500
yaassen-ho503mo4s-mostafalol1233s-projects.vercel.app
/
Traceback (most recent call last): File "/var/task/vc__handler__python.py", line 14, in <module> __vc_spec.loader.exec_module(__vc_module) File "<frozen importlib._bootstrap_external>", line 999, in exec_module File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed File "/var/task/app.py", line 1, in <module> from flask import Flask, request, jsonify, render_template, send_from_directory ModuleNotFoundError: No module named 'flask' Python process exited with exit status: 1. The logs above can help with debugging the issue.
Jul 28 18:44:52.18
GET
500
yaassen-ho503mo4s-mostafalol1233s-projects.vercel.app
/
Traceback (most recent call last): File "/var/task/vc__handler__python.py", line 14, in <module> __vc_spec.loader.exec_module(__vc_module) File "<frozen importlib._bootstrap_external>", line 999, in exec_module File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed File "/var/task/app.py", line 1, in <module> from flask import Flask, request, jsonify, render_template, send_from_directory ModuleNotFoundError: No module named 'flask' Python process exited with exit status: 1. The logs above can help with debugging the issue.
Jul 28 18:44:42.56
GET
500
yaassen-ho503mo4s-mostafalol1233s-projects.vercel.app
/
Traceback (most recent call last): File "/var/task/vc__handler__python.py", line 14, in <module> __vc_spec.loader.exec_module(__vc_module) File "<frozen importlib._bootstrap_external>", line 999, in exec_module File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed File "/var/task/app.py", line 1, in <module> from flask import Flask, request, jsonify, render_template, send_from_directory ModuleNotFoundError: No module named 'flask' Python process exited with exit status: 1. The logs above can help with debugging the issue.
Jul 28 18:44:42.38
GET
500
yaassen-ho503mo4s-mostafalol1233s-projects.vercel.app
/
Traceback (most recent call last): File "/var/task/vc__handler__python.py", line 14, in <module> __vc_spec.loader.exec_module(__vc_module) File "<frozen importlib._bootstrap_external>", line 999, in exec_module File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed File "/var/task/app.py", line 1, in <module> from flask import Flask, request, jsonify, render_template, send_from_directory ModuleNotFoundError: No module named 'flask' Python process exited with exit status: 1. The logs above can help with debugging the issue.
Jul 28 18:44:41.63
GET
500
yaassen-ho503mo4s-mostafalol1233s-projects.vercel.app
/
Traceback (most recent call last): File "/var/task/vc__handler__python.py", line 14, in <module> __vc_spec.loader.exec_module(__vc_module) File "<frozen importlib._bootstrap_external>", line 999, in exec_module File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed File "/var/task/app.py", line 1, in <module> from flask import Flask, request, jsonify, render_template, send_from_directory ModuleNotFoundError: No module named 'flask' Python process exited with exit status: 1. The logs above can help with debugging the issue.
Jul 28 18:44:41.52
GET
500
yaassen-ho503mo4s-mostafalol1233s-projects.vercel.app
/
Traceback (most recent call last): File "/var/task/vc__handler__python.py", line 14, in <module> __vc_spec.loader.exec_module(__vc_module) File "<frozen importlib._bootstrap_external>", line 999, in exec_module File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed File "/var/task/app.py", line 1, in <module> from flask import Flask, request, jsonify, render_template, send_from_directory ModuleNotFoundError: No module named 'flask' Python process exited with exit status: 1. The logs above can help with debugging the issue.
No more logs to show within selected timeline

Logs
1 Error

Traceback (most recent call last):
File "/var/task/vc__handler__python.py", line 14, in <module>
__vc_spec.loader.exec_module(__vc_module)
File "<frozen importlib._bootstrap_external>", line 999, in exec_module
File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
File "/var/task/app.py", line 1, in <module>
from flask import Flask, request, jsonify, render_template, send_from_directory
ModuleNotFoundError: No module named 'flask'
Python process exited with exit status: 1. The logs above can help with debugging the issue.
GET

/

500
Request started
Jul 28 18:44:53.59GMT+3



Function Invocation

500
Internal Server Error




